detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/Project/ProjectJsonResolver.cs

```diff
@@ -1,12 +1,10 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using Newtonsoft.Json.Linq;
```

Comment:
License of Linq?

Reply:
This library is open source under the MIT license and is free for commercial use. For details checkout the following link
https://www.newtonsoft.com/json#:~:text=Licensing,is%20free%20for%20commercial%20use.